docs/source/write_first_app.rst

```diff
@@ -101,889 +97,485 @@ bringing up a new one.
   await contract.submitTransaction('InitLedger');
-  console.log('*** Result: committed');
-
-Chaincode ``'InitLedger'`` function
-
-.. code:: bash
-
-   async InitLedger(ctx) {
-        const assets = [
-            {
-                ID: 'asset1',
-                Color: 'blue',
-                Size: 5,
-                Owner: 'Tomoko',
-                AppraisedValue: 300,
-            },
-            {
-                ID: 'asset2',
-                Color: 'red',
-                Size: 5,
-                Owner: 'Brad',
-                AppraisedValue: 400,
-            },
-            {
-                ID: 'asset3',
-                Color: 'green',
-                Size: 10,
-                Owner: 'Jin Soo',
-                AppraisedValue: 500,
-            },
-            {
-                ID: 'asset4',
-                Color: 'yellow',
-                Size: 10,
-                Owner: 'Max',
-                AppraisedValue: 600,
-            },
-            {
-                ID: 'asset5',
-                Color: 'black',
-                Size: 15,
-                Owner: 'Adriana',
-                AppraisedValue: 700,
-            },
-            {
-                ID: 'asset6',
-                Color: 'white',
-                Size: 15,
-                Owner: 'Michel',
-                AppraisedValue: 800,
-            },
-        ];
-
-        for (const asset of assets) {
-            asset.docType = 'asset';
-            await ctx.stub.putState(asset.ID, Buffer.from(JSON.stringify(asset)));
-            console.info(`Asset ${asset.ID} initialized`);
-        }
-    }
 
-The terminal output entry should look similar to below:
+An initialization function to set up initial ledger state would typically be called only once after a smart contract
+is deployed for the first time. A new version of the smart contract would likely not require the initialization to be
+performed again. In this case it is good practice for the transaction function to check and perhaps also record some
+ledger state on invocation to ensure initialization is not accidentally carried out more than once.
 
-.. code:: bash
-
-  Submit Transaction: InitLedger, function creates the initial set of assets on the ledger
-
-
-Fifth, the application invokes each of the chaincode functions
---------------------------------------------------------------
+Fifth, the application invokes transaction functions to query and modify assets
+-------------------------------------------------------------------------------
 
 First, a word about querying the ledger.
 
-Each peer in a blockchain network hosts a copy of the `ledger <./ledger/ledger.html>`_. An application
-program can view the most recent data from the ledger using read-only invocations of
-a smart contract running on your peers called a query.
+Each peer in a blockchain network hosts a copy of the `ledger <./ledger/ledger.html>`_. An application can query the
```

Comment:
"channel," if we agree to not equate the entire network with a channel. ...

Reply:
I don't see how "channel" can be correct here.

Reply:
You can check with others (@denyeart), but my view is that one blockchain "network" can contain multiple channels within it (essentially, sub-networks). In which case, each channel would host a copy of the ledger for that channel, but would not host a copy of the ledger for the whole network, as such a single thing doesn't exist. So it's just a semantics decision.

Reply:
I agree with the first sentence, although the terminology does not match that of the APIs, which this tutorial is describing. The rest makes no sense to me. The channel is not a physical thing so it can’t hold a copy of any data. Peers are physical things, and each one has a complete copy of the ledger stored on physical storage for each of the channels they belong to.

Please note that This text was already in the tutorial before my PR, and my PR has since removed this text.